qiita_pet/templates/redbiom.html

```diff
@@ -0,0 +1,192 @@
+             result = data
+          {% end %}
+            return result
+        }, targets: [1]},
+        ],
+      "language": {
+          "search": "Filter results by column data:",
+          "loadingRecords": "Please wait - loading information ...",
+          "zeroRecords": "  "
+      },
+  });
+
+  // Add event listener for opening and closing details
+  $('#redbiom-table tbody').on('click', 'div.details-control', function () {
+      var table = $('#redbiom-table').DataTable();
+      var tr = $(this).closest('tr');
+      var row = table.row( tr );
+
+      if ( row.child.isShown() ) {
+        // This row is already open - close it
+        row.child.hide();
+        tr.removeClass('shown');
+      } else {
+        // Open this row
+        // modified from: https://jsfiddle.net/8rejaL88/2/
+        tr.addClass('shown');
+        row.child('<p><center><img src="{% raw qiita_config.portal_dir %}/static/img/waiting.gif" style="display:block;margin-left: auto;margin-right: auto"/></center></p>', 'no-padding' ).show();
+        $.get('/artifact/info/', {ids: row.data().artifact_biom_ids})
+          .done(function ( data ) {
+            if (data['status']=='success') {
+              $('td', row.child()).html(format_biom_rows(data.data, row.index())).show();
+            } else {
+              bootstrapAlert('ERROR: ' + data['msg'], "danger", 10000);
+            }
+          });
+      }
+  });
+
+  $("#submitForm").submit(function(event){
+    event.preventDefault();
+
+    show_loading("redbiom-info");
+
+    var search = $("#search").val();
+    var search_on = $("#search_on").val();
+
+    $.post("{% raw qiita_config.portal_dir %}/redbiom/", {'search': search, 'search_on': search_on})
+      .done(function ( data ){
+        var redbiom_table = $('#redbiom-table').DataTable();
+        var redbiom_info = $('#redbiom-info');
+        // the next 4 lines reset the column filtering
+        var placer = $(".ps");
+        redbiom_table.column(3).search("").draw();
+        redbiom_table.clear().draw();
+        placer.empty();
+        if(data.status == "error") {
+          bootstrapAlert(data.message.replace("\n", "<br/>"), "danger");
+        } else {
+          if(data.message != ''){
+            redbiom_info.html(
+              `<div class="alert alert-warning alert-dismissible" role="alert">
+              <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <strong>Warning!</strong> ` + data.message + `</div><br/>`)
+          } else if(data.data){
+            redbiom_table.rows.add(data.data).draw();
+            redbiom_info.html('');
+          }
+        }
+      });
+    });
+  });
+</script>
+
+{%end%}
+
+{%block content%}
+  <small>
+    Redbiom only searches on public data and the data is updated nightly. Note that you will only be able to expand and add artifacts to analyses if you are signed into Qiita.
```

Comment:
Is this accurate? @wasade will it be possible to update the redis db every night?

Reply:
We have not touched load mechanisms since June, and there is work to do there, but it should be feasible

Reply:
yes, that's a discussion for next week's meeting.